src/pulse/integration/fmpapi_to_db.py

```diff
@@ -80,46 +81,51 @@ def load_historical_prices():
     #     # Get historic stock prices
         # Fetch SP500 table and get teh list of symbols
 
-        symbol = "MSFT"
-        historical_price_api = Historical_prices(symbol)
-        historical_price_json_data = historical_price_api.fetch()
+        symbols = SP500_table()
+        current_date = datetime.date.today()
+        date_str = current_date.strftime('%Y-%m-%d')
```

Comment:
You should do all the from and to date date manipulations in the internal class, not from outside.